hydpy/models/hland_v4.py

```diff
@@ -21,13 +21,12 @@
 vertically arranged storages.  The two upper storages (|BW1| and |BW2|) are
 zone-specific.  In comparison, the upper zone layer (|UZ|) of |hland_v1| is
 subbasin-specific.  For the subbasin-wide lower zone storage (|LZ|), there is no
-difference to |hland_v1|.  |hland_v4| allows for additional runoff concentration via a
-linear storage cascade (|SC|), like implemented in |hland_v2|, which is supposed to
-improve flexibility when modelling the effects of sealed areas on runoff generation.
-Setting the number of storages to one agrees with COSERO's approach to model runoff
-concentration with a single bucket.  In agreement with COSERO but in contrast to
-|hland_v2| and |hland_v3|, |hland_v4| does not send its base flow through the storage
-cascade.
+difference to |hland_v1|.  Like all |hland| application models, |hland_v4| allows an
+optional submodel for runoff concentration.  Using |rconc_nash| and setting the number
+of storages to one agrees with COSERO's approach to model runoff concentration
+with a single bucket. In agreement with COSERO but in contrast to |hland_v3|, in
+|hland_v4| base flow is not taken into account when calculating runoff concentration.
```

Comment:
Maybe a little bit clearer (active): "|hland_v4| does not take base flow into account when calculating runoff concentration."

Reply:
ok